tests/test_torch_agent.py

```diff
@@ -103,11 +42,13 @@ def get_agent(**kwargs):
 class TestTorchAgent(unittest.TestCase):
```

Comment:
you could do a the `@unittest.skipIf` around the class instead of doing it on every `def`, and it will skip all of them.